src/graphics/dom-layer.js

```diff
@@ -64,6 +64,18 @@ Crafty._registerLayerTemplate("DOM", {
         Crafty._removeDrawLayerInstance(this);
     },
 
+    _lastParentNode: null,
+    events: {
+        "Freeze":function(){
+            this._lastParentNode = this._div.parentNode;
+            this._div.parentNode.removeChild(this._div);
+        },
+        "Unfreeze":function(){
+            this._lastParentNode.appendChild(this._div);
+            delete this._lastParentNode;
+        }
+    },
```

Comment:
Yeah, looking at it now I don't think it's necessary.  I think I was recalling a past era when drawing was triggered through an event, rather than a direct method call.